packages/loot-core/src/client/budgets/budgetsSlice.ts

```
@@ -0,0 +1,589 @@
+      }),
+    );
+
+    const { id, error } = await send('download-budget', {
+      fileId: cloudFileId,
+      replace,
+    });
+
+    if (error) {
+      if (error.reason === 'decrypt-failure') {
+        const opts = {
+          hasExistingKey: error.meta && error.meta.isMissingKey,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)